dataprofiler/profilers/helpers/report_helpers.py

```diff
@@ -100,7 +100,68 @@ def _prepare_report(report, output_format=None, omit_keys=None):
         if isinstance(value, set):
             value = sorted(list(value))
 
-        if isinstance(value, dict):
+        if "*" in omit_keys:
+            continue
+
+        # For data_stats (in structured case), need to recurse through a list
+        elif key == "data_stats" and isinstance(value, list) \
+                and "data_stats" not in omit_keys:
+
+            # Map cols to keep in report with their layer's omit keys
+            kept_cols_oks = dict()
+            # Cols that will have None in report since they were omitted
```

Comment:
i suggest having a space between the line and the next comment, below too

Reply:
Whoops, at one point this was too long but I made some changes and never made it back to 1 line.